src/components/Card/components/CardDescription.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC } from 'react'
+
+export interface CardDescriptionProps {
+    description: string
+}
+
+const CardDescription: FC<CardDescriptionProps> = ({
+    description,
+}): JSX.Element => {
+    return (
+        <p
+            itemProp='description'
+            dangerouslySetInnerHTML={{ __html: description }}
```

Comment:
Now this is a new component, we should change dangerouslySetInnerHTML to children.

E.G.: `<p itemProp='description'>{children}</p>`

Reply:
Fixed